src/components/my-journey/Journey.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`Line 43` I feel that it would be better if the skeleton code were extracted as a component. It is kind of confusing to see div tags without any children or text inside them.

Reply:
`Line 143` and `Line 148` change the variable name to `IsStepInprogress` and `isActive` since they are booleans

Reply:
`Line 160 , 173 ,178` It would be better to change this part of the code` subStep.subStepName === "Apply to the program" ` to another type of check, as missing type may lead to dysfunctionality.

At least it can be placed at the top and reused in all three of them

Reply:
`Line 158` For consistency, it is better to use the `&&` operator instead of using an if block.

Reply:
`Line 195` Remove `{" "}`